### ISSUE TITLE
Log format need add to log module

### ISSUE THREAD

**Issue by zloidemon** [tarantool/doc#604]
I think we have to move this into log module or copy-paste there from configuration section.

```
 log_format¶

    Log entries have two possible formats:

        ‘plain’ (the default), or
        ‘json’ (with more detail and with JSON labels).

    Here is what a log entry looks like after box.cfg{log_format='plain'}:
```

**Comment by lenkis** [comment]
Taken from https://www.tarantool.io/en/doc/1.7/reference/configuration/#confval-log_format
Please duplicate (copy-paste) into https://www.tarantool.io/en/doc/1.7/reference/reference_lua/log/ and possibly cross-link the two sections.

**Comment by sofiayem** [comment]
Fixed by #629 PR.
